src/DocumentFragment.js

```diff
@@ -0,0 +1,71 @@
+  // ### DocumentFragment Shim
+  //
+  // A representation of a document fragment that can be normally generated
+  // by calling `document.createDocumentFragment`. Since the document fragment
+  // is itself a node, it inherits all properties of a node.
+module.exports = (function(){
+  "use strict";
+
+  var Node = require('./Node');
+  var voidHtmlElements = ["area", "base", "br", "col", "command", "embed", "hr",
```

Comment:
let's not repeat ourselves: that list already exists in https://github.com/mozilla/slowparse/blob/gh-pages/src/HTMLParser.js#L111 so if you want it duplicated, that needs to be refactored as a file that has `module.exports = [....]` so you can load it in as `voidHTMLElements = require("...../voidElements")`